### PR TITLE
Do some date triage for 2021-11

### PR DESCRIPTION
**Pull request by camelid**
- Ensure date-check cron job is using latest stable Rust
- Update some date-check comments

### CODE REVIEW

File: src/rustdoc-internals.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ these passes, please let us know!)
 
 [44136]: https://github.com/rust-lang/rust/issues/44136
 
-Here is the list of passes as of <!-- date: 2021-02 --> February 2021:
+Here is the list of passes as of <!-- date: 2021-10 --> October 2021:
```
Comment (camelid) [inline src/rustdoc-internals.md:69]: I updated this section last month but forgot to change this date-check comment.

File: src/overview.md [hunk 2]
```diff
@@ -262,7 +262,7 @@ Moreover, the compiler wasn't originally built to use a query system; the query
 system has been retrofitted into the compiler, so parts of it are not query-fied
 yet. Also, LLVM isn't our code, so that isn't querified either. The plan is to
 eventually query-fy all of the steps listed in the previous section,
-but as of <!-- date: 2021-02 --> February 2021, only the steps between HIR and
+but as of <!-- date: 2021-11 --> February 2021, only the steps between HIR and
```
Comment (JohnTitor) [inline src/overview.md:265]: I think this should be:

```suggestion
but as of <!-- date: 2021-11 --> November 2021, only the steps between HIR and
```

**Review by JohnTitor — APPROVED** [review]
Overall LGTM with one nit.

File: src/overview.md (hunk 2 above)
Comment (camelid) [reply src/overview.md:265]: Thanks! 🤦

**Pull request merged**